### PR TITLE
Added a page encouraging researching and the purpose of community mentor

### PR DESCRIPTION
**Pull request by CMNatic**

### CODE REVIEW

File: docs/general/researching-and-asking-for-help.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+Greatly formatted questions are both really helpful in exchanges between users. The Discord can be slow at times, so formatting your question similar to the one above skips the process of trying to figure out where you're stuck and on what room.
+
+### Mentors
+A fantastic attribute to TryHackMe is the community itself. Other users like to help each other out - and it's wonderful. We really encourage users to continue helping each other out.
+
+However, we can't solely rely on users to be there 24/7 for each other, nor validate what they are saying. TryHackMe has users who throughout time have consistently guided other users through issues they experience...from technical issues relating to connecting to the network to providing resources to help resolve challenges.
+
+**These Mentors, much like Moderators are not paid** and are users of the TryHackMe platform such as yourself! Because of this, please be patient. If your answer isn't answered by any user - someone will get too it as and when they can! There is no need to post your issue in every and all channels.
```
Comment (Kent55) [inline docs/general/researching-and-asking-for-help.md]: Only one issue I noticed here is that `"too"` should be `"to"` in the sentence `"someone will get too it as and when they can!"`

Comment (CMNatic) [reply docs/general/researching-and-asking-for-help.md]: Thanks for noticing that! Apologies for the oversight :)

Comment (Kent55) [reply docs/general/researching-and-asking-for-help.md]: Did you want to put in the change? @CMNatic

Comment (CMNatic) [reply docs/general/researching-and-asking-for-help.md]: Sure, I'll change and open as a new PR @Kent55

Comment (Kent55) [reply docs/general/researching-and-asking-for-help.md]: > Thanks for noticing that! Apologies for the oversight :)
> 
> Shall I fix typo and open as new PR? Or are you able to resolve and merge directly?

^^^ yes please :)

**Pull request closed without merging**